js/modules/tamagotchi.js

```diff
@@ -27,6 +54,58 @@ export default class Tamagotchi {
 		const displayElement = document.querySelector(elementSelector);
 		displayElement.innerText = this.fun.value;
 	};
+	displayParagraph = (textContent) => {
+		const paragraph = document.querySelector('.tamagotchiStates');
+		paragraph.textContent = textContent;
+	};
+	displayState = (imageSelector) => {
+		const image = document.querySelector(imageSelector);
+		if (
+			this.energy.value &&
```

Comment:
**nitpick:** You don't have any single which your Tamagotchi currently have. Doing it this way will probably overwrite your actions, and it will make impossible to control. Here is good example - how (based on your current implementation) would you check what state your Tamagotchi currently have? You would have to check CSS class - but it's out of JS logic, which is a very bad idea. You don't have any control over it.

Try to implement something like: `this.activeState`, and then you could just do simple `switch` which could look like eg.:

```
switch(this.currentState) {
        case this.states.sleepy.stateName:
           image.src = this.states.sleepy.path;
	   this.displayParagraph(this.states.sleepy.stateName);
           break;
}
           ...
```